modules/electrophysiology_uploader/jsx/UploadForm.js

```diff
@@ -197,17 +206,20 @@ export default function UploadForm(props) {
         error = resp.error;
       }
     } else if (xhr.status == 0) {
-      error = 'Upload failed: a network error occured';
+      error = t('Upload failed: a network error occured',
+        {ns: 'electrophysiology_uploader'});
     } else if (xhr.status == 413) {
-      error = 'Please make sure files are not larger than '
-          + props.maxUploadSize;
+      error = t('Please make sure files are not larger than',
+        {ns: 'electrophysiology_uploader'})
```

Comment:
This should be a template string.

Imaging_uploader already has `msgid "Please make sure files are not larger than {{maxFileSize}}"`

so maybe that should be moved to the loris namespace and used.